### PR TITLE
Remove extra full stop from function header of `get_operator`

### PR DESCRIPTION
**Pull request by hansrajdas**

### CODE REVIEW

File: Python/ast.c [hunk 1]
```diff
@@ -958,7 +958,7 @@ PyAST_FromNode(const node *n, PyCompilerFlags *flags, const char *filename_str,
 
 }
 
-/* Return the AST repr. of the operator represented as syntax (|, ^, etc.)
+/* Return the AST repr of the operator represented as syntax (|, ^, etc.)
```
Comment (brandtbucher) [inline Python/ast.c]: Pretty sure "repr." here is short for "representation". This function has nothing to do with Python's `repr()`/`__repr__()`...

**Review by brandtbucher — CHANGES_REQUESTED** [review]
Not sure about this one @hansrajdas. See below:

File: Python/ast.c (hunk 1 above)
Comment (hansrajdas) [reply Python/ast.c]: - Agreed, repr expanded to representation
- Extra line between function header and function body also deleted

**Comment by brandtbucher** [comment]
I personally think that the comment (and whitespace) is fine as-is. Sorry.

**Comment by hansrajdas** [comment]
> I personally think that the comment (and whitespace) is fine as-is. Sorry.

Ok, Shall we close this PR?

**Comment by brandtbucher** [comment]
Yes, I would recommend closing it.

**Comment by hansrajdas** [comment]
> Yes, I would recommend closing it.

Sure, Thank you!

**Pull request closed without merging**